angular/src/lib/select/tests/select.component.spec.ts

```diff
@@ -136,315 +138,8 @@ describe('SelectComponent', () => {
     flush();
 
     expect(selectComponent.overlayOpen).toBe(true);
-    expect(overlayContainerElement.textContent.trim()).toContain('first option');
-    expect(overlayContainerElement.textContent.trim()).toContain('second option');
   }));
```

Comment:
We need more test cases .. all this does is make sure it creates?  What about selecting the item .. what about the multi select?

Reply:
Will add more tests